CHANGELOG.md

```diff
@@ -4,7 +4,32 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/0.6.2...HEAD)
+#### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...HEAD)
```

Comment:
I think we're missing a v1 header here? I believe this should be:

`#### [1.0.0](https://github.com/googlemaps/android-maps-utils/compare/0.6.2...1.0.0)`

...and then we should have another "Unreleased" section above it that would be:

`#### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...HEAD)`